rendered/_config.yml

```diff
@@ -1,3 +1,5 @@
 theme: jekyll-theme-tactile
 url: "https://zips.z.cash"
 markdown: GFM
+gems:
+  - jekyll-redirect-from
```

Comment:
This is unnecessary (and will not work). It would only have an effect when rendering the HTML; see comment below.

Reply:
If this doesn't work, then the entire `_config.yml` is doing nothing, because that config file is for Jinja.

Reply:
https://github.com/orgs/community/discussions/53549 seems to indicate that Jinja is _always_ running because GitHub Pages is powered by it, which would imply that gems _do_ work.